test/unit/service/get-service.test.js

```diff
@@ -0,0 +1,104 @@
+import { beforeEach, afterEach, describe, it } from 'mocha';
```

Comment:
Can you please add unit tests for operation service class. We should cover methods:
getResponsesStatuses
validateAssertion